plothist/plotters.py

```diff
@@ -469,107 +321,237 @@ def plot_error_hist(hist, ax, **kwargs):
     )
 
 
-def plot_hist_difference(hist1, hist2, ax, **kwargs):
-    """
-    Plot the difference between two histograms.
-
-    Parameters
-    ----------
-    hist1, hist2 : boost_histogram.Histogram
-        The histograms for which the difference hist1 - hist2 will be plotted.
-    ax : matplotlib.axes.Axes
-        The Axes instance for plotting.
-    **kwargs
-        Additional keyword arguments forwarded to ax.errorbar().
-    """
-    difference = hist1.values() - hist2.values()
-    difference_uncertainty = np.sqrt(hist1.variances() + hist2.variances())
-    ax.errorbar(
-        x=hist1.axes[0].centers,
-        xerr=None,
-        y=difference,
-        yerr=difference_uncertainty,
-        fmt=".",
-        **kwargs,
-    )
-
-
 def compare_two_hist(
-    hist_1, hist_2, xlabel=None, ylabel=None, x1_label="x1", x2_label="x2", save_as=None
+    hist_1,
+    hist_2,
+    xlabel=None,
+    ylabel=None,
+    x1_label="x1",
+    x2_label="x2",
+    comparison="ratio",
+    comparison_ylim=None,
+    ratio_uncertainty="uncorrelated",
+    save_as=None,
+    fig=None,
+    ax_main=None,
+    ax_comparison=None,
 ):
     """
     Compare two histograms.
 
     Parameters
     ----------
-    hist_1, hist_2 : boost_histogram.Histogram
-        The histograms to be compared.
+    hist_1 : boost_histogram.Histogram
+        The first histogram to compare.
+    hist_2 : boost_histogram.Histogram
+        The second histogram to compare.
     xlabel : str, optional
-        The label for the x-axis of the comparison plot.
+        The label for the x-axis. Default is None.
     ylabel : str, optional
-        The label for the y-axis of the comparison plot.
-    x1_label, x2_label : str, optional
-        The labels for the two histograms being compared.
-    save_as : str, optional
-        If provided, the filename to save the figure as.
+        The label for the y-axis. Default is None.
+    x1_label : str, optional
+        The label for the first histogram. Default is "x1".
+    x2_label : str, optional
+        The label for the second histogram. Default is "x2".
+    comparison : str, optional
+        The type of comparison to plot. Default is "ratio".
```

Comment:
We should tell the user to look at `plot_comparison` for details. Also, `ratio_uncertainty` is missing

Reply:
Already in the "See Also" section